### ISSUE TITLE
Change Repo from Public to Private

### ISSUE THREAD

**Issue by barricade823** [lukeb2014/cobblestone#1]
## I would prefer that since we are going to be sharing code and ideas through GitHub that the repository be private.

@lukeb2014 If you currently don't have access to private repos, the ownership of the repository could be transferred to me as I have a paid account with unlimited private repos. Your call though on whether or not you and your team want ownership of the repo, but either way I think it needs to be private.

**Comment by lukeb2014** [comment]
@barricade823 I completely agree, I'll talk with Ryan and see what he says about it being either yours or ours.
